modules/certificate/certificate.graphql.js

```diff
@@ -73,6 +73,7 @@ const certificateTypes = `
     deleteCertificate(id: ID!): CertificateResult
     generateCertificate (newCertificates: [GenerateCertificateInput]!, email: String, dateStart: Date ): CertificatesResult
     updateCertificate(name: String!): CertificateResult
+	 gitftCertificateToEmail(id: ID!, email: String!, oldEmail: String!, language: Int!): CertificateResult
```

Comment:
Please fix the naming of `gitftCertificateToEmail` to `giftCertificateToEmail`

Reply:
done